code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -789,6 +789,17 @@
 				emotelist += "\nMachine specific emotes :- beep(s)-(none)/mob, buzz(es)-none/mob, no-(none)/mob, ping(s)-(none)/mob, yes-(none)/mob"
 			else if(species.name == "Slime People")
 				emotelist += "\nSlime people specific emotes :- squish(es)-(none)/mob"
+			else if(species.name == "Drask")
+				emotelist += "\nDrask specific emotes :- drone(s)-(none)/mob, hum(s)-(none)/mob, rumble(s)-(none)/mob"
+			else if(species.name == "Kidan")
+				emotelist += "\nKidan specific emotes :- click(s), clack(s)"
+			else if(species.name == "Unathi")
+				emotelist += "\nUnathi specific emotes :- hiss(es)"
+			if (!(species.name == "Slime People"))
+				for(var/obj/item/organ/external/L in bodyparts) // if your limbs are squishy you can squish too!
```

Comment:
could just stick this bit under an else under the slimepeople check already, rather than checking if they are a slime person in 791, then checking again. Might as well just use the single check.